etc/greenboot/green.d/00_grub_fallback.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+# We need to determine whether this boot was a fallback boot
+if grub2-editenv list | grep -q "fallback_active=true"; then
+  grub2-editenv - unset fallback_active
+  grub2-editenv - set boot_success=0 # this should already be 0
+  grub2-editenv - set boot_counter=2
+  rpm-ostree rollback --reboot
+else
+  grub2-editenv - set boot_success=1
```

Comment:
So in fact, it seems like we're doing this either way -- so we can factor that out of the if statement.